spec/models/stat/anime_activity_history_spec.rb

```diff
@@ -0,0 +1,76 @@
+# rubocop:disable Metrics/LineLength
```

Comment:
There's no reason to comment out code — your ~mistake~ _code_ is forever recorded in the git history, where we can recover it 😄